stock_warehouse_calendar_orderpoint/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Stock Warehouse Calendar (reordering rules)",
+    "summary": "Adds a calendar to the Warehouse for reordering rules",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
```

Comment:
I think it's AGPL seeing files headers

Reply:
Bad code snippet, all have been fixed now I guess.

Reply:
And it has to be LGPL as the dependency `stock_warehouse_calendar` is LGPL.